src/Tests/Microsoft.NET.TestFramework/AspNetSdkTest.cs

```diff
@@ -18,10 +18,9 @@ public abstract class AspNetSdkTest : SdkTest
     {
         public readonly string DefaultTfm;
 
-        protected AspNetSdkTest(ITestOutputHelper log) : base(log)
+        protected AspNetSdkTest(ITestOutputHelper log, Assembly testAssembly) : base(log)
```

Comment:
Can we use GetType() here instead?  We do that in some other types in the framework like [MemoryStream](https://github.com/dotnet/runtime/blob/6ca8c9bc0c4a5fc1082c690b6768ab3be8761b11/src/libraries/System.Private.CoreLib/src/System/IO/MemoryStream.cs#L352).

Reply:
Since this class is not concrete, GetType() wouldn't work here.